tee-worker/litentry/core/data-providers/src/graphql.rs

```diff
@@ -212,4 +293,21 @@ mod tests {
 			assert!(false);
 		}
 	}
+
+	#[test]
+	fn verified_credentials_total_transactions_work() {
+		let query = VerifiedCredentialsTotalTxs {
+			addresses: vec!["EGP7XztdTosm1EmaATZVMjSWujGEj9nNidhjqA2zZtttkFg".to_string()],
+			networks: vec![
+				VerifiedCredentialsNetwork::Kusama,
+				VerifiedCredentialsNetwork::Polkadot,
+			],
+		};
+		let mut client = GraphQLClient::new();
+		if let Ok(r) = client.query_total_transactions(query) {
+			assert!(r.get(0).unwrap().total_transactions >= 41)
+		} else {
+			assert!(false)
+		}
```

Comment:
how about this?
```
let r = client.query_total_transactions(query);
assert!(r.is_ok());
assert!(r.unwrap().get(0).unwrap().total_transactions > 41);
```

Reply:
Maybe we should also assert that the list is not empty.